grand_challenge_forge/schemas.py

```diff
@@ -83,3 +83,41 @@ def validate_pack_context(context):
         raise InvalidContextError(
             f"Invalid pack context provided:\n'{truncate_with_epsilons(context)!r}'"
         ) from e
+
+
+ALGORITHM_TEMPLATE_CONTEXT_SCHEMA = {
+    "type": "object",
+    "properties": {
+        "algorithm": {
+            "type": "object",
+            "properties": {
+                "title": {"type": "string"},
+                "url": {"type": "string"},
+                "inputs": {
+                    "type": "array",
+                    "items": COMPONENT_INTERFACE_SCHEMA,
+                },
+                "outputs": {
+                    "type": "array",
+                    "items": COMPONENT_INTERFACE_SCHEMA,
+                },
+            },
+            "required": ["title", "url", "slug", "inputs", "outputs"],
```

Comment:
Is the title still required now that you require the slug?

Reply:
Yes, I think it is still good to add a title.